aqua/qt6/Portfile

```diff
@@ -137,12 +137,12 @@ array set modules {
             d9924d6fd4fa5f8e24458c87f73ef3dfc1e7c9b877a5407c040d89e6736e2634
             46541252
         }
-        "port:ninja port:pkgconfig"
-        "port:assimp port:brotli path:bin/cmake:cmake path:bin/dbus-daemon:dbus port:double-conversion port:freetype
+        "path:bin/cmake:cmake port:ninja port:pkgconfig"
+        "port:brotli path:bin/dbus-daemon:dbus port:double-conversion port:freetype
             path:lib/pkgconfig/glib-2.0.pc:glib2 path:lib/pkgconfig/harfbuzz.pc:harfbuzz
-            port:hunspell path:lib/pkgconfig/icu-uc.pc:icu port:jasper path:include/turbojpeg.h:libjpeg-turbo port:libb2
-            port:libiconv port:libpng port:md4c path:bin/node:nodejs16 path:lib/pkgconfig/libpcre2-posix.pc:pcre2
-            port:tiff port:webp port:zlib path:lib/pkgconfig/libzstd.pc:zstd"
+            path:lib/pkgconfig/icu-uc.pc:icu path:include/turbojpeg.h:libjpeg-turbo port:libb2
+            port:libiconv port:libpng port:md4c path:lib/pkgconfig/libpcre2-posix.pc:pcre2
+            port:zlib path:lib/pkgconfig/libzstd.pc:zstd"
```

Comment:
so did you then add these dependencies in the other subports that you say need them?

Reply:
They're already there, no clue why they were duplicated over qtbase
https://github.com/macports/macports-ports/blob/28a14b0a637aa68694e30545662b2c09af683ffa/aqua/qt6/Portfile#L207

Reply:
zlib is included to link off of the macports zlib lib

`qtbase-everywhere-src-6.2.4 % cat config_help.txt|grep zlib
  -zlib ................ Select used zlib [system/qt]
`

Reply:
ZSTD is there as it's an optional package that cmake picks up

`qtbase-everywhere-src-6.2.4 % ls cmake/FindZSTD.cmake `

Reply:
I believe webp and tiff can probably be removed.  I can't find anything in qt6-base that searches for them.

Reply:
zlib and zstd aren't removed?  They're right there...

Reply:
@TellowKrinkle  - Apologies again, missed the line at the end